code/modules/projectiles/gun_system.dm

```diff
@@ -616,13 +650,22 @@
 	if(!.)
 		return FALSE
 
+	setup_bullet_accuracy()
 	user.remove_movespeed_modifier(MOVESPEED_ID_AIM_SLOWDOWN)
 
 	if(HAS_TRAIT(src, TRAIT_GUN_IS_AIMING))
 		toggle_aim_mode(user)
 
 	return TRUE
 
+/obj/item/weapon/gun/toggle_wielded(user, new_value)
+	switch(new_value)
+		if(null)
+			flags_item ^= WIELDED
+		if(FALSE)
+			flags_item &= ~(WIELDED|FULLY_WIELDED)
+		if(TRUE)
+			flags_item |= WIELDED
```

Comment:
why is this using bools and null while not treating null as 0

Reply:
Mentioned in discord, the null if from the base proc. As far as I understand it is unused and doesn't do anything, but I didn't want to delete it if I was missing something.